core/win32/os_private.h

```diff
@@ -471,11 +471,11 @@ extern uint context_xstate;
 
 #define XSTATE_HEADER_SIZE 0x40 /* 512 bits */
 #define YMMH_AREA(ymmh_area, i) (((dr_xmm_t *)ymmh_area)[i])
-#define MAX_CONTEXT_64_SIZE 0x6ef /* as observed on win10-x64 */
+#define MAX_CONTEXT_64_SIZE 0xd2f /* as observed on win10-x64 */
 #ifdef X64
 #    define MAX_CONTEXT_SIZE MAX_CONTEXT_64_SIZE
 #else
-#    define MAX_CONTEXT_SIZE 0x4e3 /* as observed on win10-x64 */
+#    define MAX_CONTEXT_SIZE 0xd2f /* as observed on win10-x64 */
```

Comment:
Surprising that 32-bit is just as big.  The only potential problem is that these structures are on the stack in a number of routines: so the risk is stack overflow with DR's tiny stacks.  But if the tests pass it should be ok.  Long-term we may need to move all CONTEXTs to the heap.